kvmagent/kvmagent/test/test_usb.py

```
@@ -0,0 +1,38 @@
+        for _ in range(6):
+            assert usb_manager.request_slot(2) == 1
+
+        # test usb version 2, controller 2
+        for _ in range(2):
+            assert usb_manager.request_slot(2) == 2
+
+        # test usb version 3, controller 2
+        assert usb_manager.request_slot(3) == 2
+
+        # insufficient usb controllers
+        with pytest.raises(Exception):
```

Ruff (0.12.2)

Do not assert blind exception: `Exception`

(B017)